src/backend/src/services/tasks.services.ts

```diff
@@ -149,12 +160,16 @@ export default class TasksService {
     const originalTask = await prisma.task.findUnique({
       where: { taskId },
       include: {
-        wbsElement: { include: { project: { ...projectQueryArgs } } }
+        wbsElement: { include: { project: { ...projectQueryArgs } } },
+        assignees: true
       }
     });
     if (!originalTask) throw new NotFoundException('Task', taskId);
     if (originalTask.dateDeleted) throw new DeletedException('Task', taskId);
 
+    const originalAssignees = originalTask.assignees.map((user) => user.userId);
```

Comment:
can you change user to useId here, more precise

Reply:
I changed user to assignee to be more precise, but in this case the original array is users

Reply:
you're right, I was tripping, good job changing the names that actually needed changing lol